pkg/controller/operator.go

```diff
@@ -144,42 +143,48 @@ func NewKubernikusOperator(options *KubernikusOperatorOptions, logger log.Logger
 		options.AuthDomain,
 		options.AuthProject,
 		options.AuthProjectDomain,
+		logger,
 	)
 
-	o.Clients.Satellites = kube.NewSharedClientFactory(secrets, klusters)
+	o.Clients.Satellites = kube.NewSharedClientFactory(secrets, klusters, logger)
 
 	// Add kubernikus types to the default Kubernetes Scheme so events can be
 	// logged for those types.
 	v1.AddToScheme(scheme.Scheme)
 	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartLogging(glog.Infof)
+	eventBroadcaster.StartLogging(func(format string, args ...interface{}) {
+		o.Logger.Log("msg", fmt.Sprint(format, args), "v", 2)
```

Comment:
We could potentially log events more structured as the arguments always mean the same. We should at least some key that allows us to filter for events.
I'm also fine with leaving this for a follow up.

Reply:
Much better.

```
ts=2017-12-18T15:06:19.787737058Z caller=github.com/sapcc/kubernikus/pkg/controller/operator.go:156 
controller=operator 
resource=event 
msg="Successfully created node 2657e86b-be51-4e5c-a5a5-dce9c5431661"
reason=SuccessfullCreateNode 
type=Normal 
kind=Kluster 
namespace=kubernikus-michi 
name=test2-5d725ddf97664a16b011e8a8dd75772b 
v=2
```

Reply:
Much better.

```
ts=2017-12-18T15:06:19.787737058Z caller=github.com/sapcc/kubernikus/pkg/controller/operator.go:156 
controller=operator 
resource=event 
msg="Successfully created node 2657e86b-be51-4e5c-a5a5-dce9c5431661"
reason=SuccessfullCreateNode 
type=Normal 
kind=Kluster 
namespace=kubernikus-michi 
name=test2-5d725ddf97664a16b011e8a8dd75772b 
v=2
```